### PR TITLE
Update project folders structure and dependencies version

### PR DESCRIPTION
**Pull request by smaniotto**

### CODE REVIEW

**Review by dleitee — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-pyflakes==1.5.0
-psycopg2==2.6.2
-pyparsing==2.1.10
-six==1.10.0
-django-environ==0.4.3
+# Lint
+flake8==3.5.0
+
+# Django
+Django==2.0.1
```
Comment (rlander) [inline requirements.txt:5]: Should we really be using 2.0 as default? I've hit a few compat issues with 2.0

**Review by rlander — CHANGES_REQUESTED** [review]
See inline comment

File: requirements.txt (hunk 1 above)
Comment (smaniotto) [reply requirements.txt:5]: Not sure yet, but they release as the last stable. I didn't see any major advantages other than simplifying a lot the URLs definition, but it's going to be a pain to upgrade if we need anything from 2.0. Do you remember any of these compatibility issues?

**Review by rlander — APPROVED** [review]

**Comment by rlander** [comment]
Sorry, didn't mean to block the PR, just to add an observation... Fixed!

File: requirements.txt (hunk 1 above)
Comment (jonatasbaldin) [reply requirements.txt:5]: I think you should embrace the future! Embrace Django 2.0!

**Pull request merged**